packages/server/src/queries/blockaid/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from "./transaction";
+
+export const BASE_URL = process.env.BLOCKAID_BASE_URL;
```

Comment:
Could we access all process env vars in the same place in the root env file of this package?